<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <configSections>
    <section name="categoriesConfigs" type="ExceptionLogParser.CategoriesConfigsSection, ExceptionLogParser" />
  </configSections>
  <startup>
    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
  </startup>
  <appSettings>
    <add key="SiteName" value="Test Site" />
    <add key="FileFolderLocation" value="C:\Users\Andrew\Documents\AlertWatch Dropbox" />
    <add key="MinimumEntriesInSegment" value="3" />
    <add key="MaximumTimeInSegment" value="120" />
    <add key="SMTPHost" value="smtp.gmail.com" />
    <add key="SMTPPort" value="587" />
    <add key="SenderAccountUserName" value="" />
    <add key="SenderAccountPassword" value="" />
    <add key="FromAddress" value="andrew.radecki@gmail.com" />
    <add key="SuccessEmail" value="andrew.radecki@gmail.com" />
    <add key="ErrorEmail" value="andrew.radecki@gmail.com" />
  </appSettings>
  <categoriesConfigs>
    <categories>
      <add plaintext="System.InvalidOperationException: There is already an open DataReader associated with this Command which must be closed first." rawtext="System.InvalidOperationException: There is already an open DataReader associated with this Command which must be closed first." />
      <add plaintext="System.InvalidOperationException: ExecuteReader requires an open and available Connection. The connection's current state is closed." rawtext="System.InvalidOperationException: ExecuteReader requires an open and available Connection. The connection's current state is closed." />
      <add plaintext="System.IO.IOException: The process cannot access the file 'c:\logs\AWOR.BJCAWORT01.AlertWatch.Beta-20180221.log' because it is being used by another process." rawtext="System.IO.IOException: The process cannot access the file 'c:\logs\AWOR.BJCAWORT01.AlertWatch.Beta-20180221.log' because it is being used by another process." />
    </categories>
  </categoriesConfigs>
</configuration>